test/input_gen/recorder.py

```diff
@@ -72,7 +72,8 @@ def _rand_like(tensorOrShape, scale=1):
     except AttributeError:
         shape = tensorOrShape
 
-    t = np.random.randint(-10, 10, shape).astype(dtype=np.float32)
+    # for relu based models, range of 0 to x is better than -x to x
+    t = np.random.randint(0, 10, shape).astype(dtype=np.float32)
```

Comment:
Should this be the case for others? we will not get less change to test if certain value is negative, eg) leaky relu.

Reply:
true. I think input range must be chosen based on model to model. I will update it as a separate PR to [0,10] for resnet and [-10,10] as default.